<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KISStester</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="recentScroll">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="recentContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>866</width>
         <height>332</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="recentLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>279</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="queueScroll">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="queueContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>866</width>
         <height>331</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="queueLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>278</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFremantle">
    <property name="title">
     <string>Fremantle</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionSettings"/>
    <addaction name="actionReload_repository_data"/>
    <addaction name="actionCommentList"/>
    <addaction name="actionTesterList"/>
    <addaction name="actionFilter_packages"/>
   </widget>
   <addaction name="menuFremantle"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Login settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionReload_repository_data">
   <property name="text">
    <string>Reload repository data</string>
   </property>
  </action>
  <action name="actionCommentList">
   <property name="text">
    <string>Package comment archive</string>
   </property>
  </action>
  <action name="actionTesterList">
   <property name="text">
    <string>Join the testing squad</string>
   </property>
  </action>
  <action name="actionFilter_packages">
   <property name="text">
    <string>Package filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
